bugrap/src/main/java/org/vaadin/bugrap/ui/ReportDetailView.java

```diff
@@ -0,0 +1,173 @@
+package org.vaadin.bugrap.ui;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+import org.vaadin.bugrap.BaseModel;
+import org.vaadin.bugrap.domain.entities.Comment;
+import org.vaadin.bugrap.domain.entities.Report;
+import org.vaadin.bugrap.ui.generated.ReportDetailViewBase;
+
+import com.vaadin.data.Binder;
+import com.vaadin.server.FileResource;
+import com.vaadin.ui.HorizontalLayout;
+import com.vaadin.ui.Link;
+import com.vaadin.ui.Notification;
+import com.vaadin.ui.Notification.Type;
+import com.vaadin.ui.ProgressBar;
+import com.vaadin.ui.Upload.ProgressListener;
+import com.vaadin.ui.Upload.Receiver;
+import com.vaadin.ui.Upload.StartedEvent;
+import com.vaadin.ui.Upload.StartedListener;
+import com.vaadin.ui.Upload.SucceededEvent;
+import com.vaadin.ui.Upload.SucceededListener;
+import com.vaadin.ui.VerticalLayout;
+import com.vaadin.ui.Window;
+
+public class ReportDetailView extends ReportDetailViewBase implements Receiver, SucceededListener, StartedListener, ProgressListener{
```

Comment:
There are some formatting issues in the file. Consider formatting the whole file.

Reply:
done